README.md

```diff
@@ -32,4 +32,7 @@ resbundler.js --assembly dynamic --compiled compiled --resDir resources --outDir
 Release Notes
 -------------
 ### 1.0.0
-Intial release
\ No newline at end of file
+Intial release
+
+### 1.0.1
```

Comment:
The custom I have been following in the other projects it to put the most recent release notes at the top.

Reply:
I applied all of your comments